webpack/JobWizard/steps/HostsAndInputs/buildHostQuery.js

```diff
@@ -13,11 +13,12 @@ export const buildHostQuery = (selected, search) => {
     hostGroups.length ? hostGroupsSearch : false,
     search.length ? search : false,
   ].filter(Boolean);
+
   if (queryParts.length === 0) {
-    return null;
+    return '1=0';
```

Comment:
The `1=0` idea is a good one, but sadly it will not work in this context. `Host.where('1 = 0')` works, because that gets translated literally to SQL, but the search query goes through scoped search first. Running this should be giving you errors such as this

```
> Host.search_for('1=0')
ScopedSearch::QueryNotSupported: Field '1' not recognized for searching!
```

Reply:
`/api/hosts?search="1=0"` gives me zero results. can also change it into `name = a AND name = b` as a hack Marek suggested

Reply:
Huh, that's odd
```shell
$ curl -u admin:changeme http://localhost:3000/api/hosts?search="1=0"
{
  "error": {"message":"Field '1' not recognized for searching!","class":"ScopedSearch::QueryNotSupported"}
}
```

Reply:
Changed so we wont have this mismatch in error/no error